sw/device/lib/dif/BUILD

```diff
@@ -0,0 +1,34 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+cc_library(
+  name = "clkmgr",
```

Comment:
this library should be statically linked

Reply:
It will be when linked into a cc_binary.  For cc_library rules, `linkstatic` determines whether a library is permitted to be statically-linked, dynamically-linked or both.

For `cc_binary` rules, `linkstatic` determines the linking type used when constructing the binary.

IIRC, the default for `cc_binary` is to static link, but dynamic linking is allowed for `cc_test` rules.